extensions/amp-story/1.0/amp-story.css

```diff
@@ -261,6 +261,7 @@ amp-story-cta-layer {
 
 /** Grid level */
 amp-story-grid-layer {
+  box-sizing: border-box !important;
```

Comment:
Why is this needed?

Reply:
If we don't include it, the `thirds` template won't consider the padding for the 100% height. See screenshots below:

No `box-sizing: border-box`:
![image](https://user-images.githubusercontent.com/5449100/48563891-5826ba80-e8c3-11e8-99b8-3c9b68dbb30d.png)

With `box-sizing: border-box`:
![image](https://user-images.githubusercontent.com/5449100/48563904-5ceb6e80-e8c3-11e8-990e-f127392a6c50.png)


You can also see this in [a Percy build](url): (the right one has`box-sizing: border-box`). And I think this is what we want, is that correct @newmuis?

Reply:
Yes, I believe the `box-sizing` is necessary here.